### PR TITLE
runtimetest: add NoNewPriviledges validation

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

File: cmd/runtimetest/main.go [hunk 1]
```diff
@@ -97,6 +97,17 @@ func validateProcess(spec *rspec.Spec) error {
 		}
 	}
 
+	ret, _, errno := syscall.Syscall6(syscall.SYS_PRCTL, 39, 0, 0, 0, 0, 0)
```
Comment (mrunalp) [inline cmd/runtimetest/main.go]: Can we define a constant for 39 and then use it?

Comment (Mashimiao) [inline cmd/runtimetest/main.go]: sure, updated

File: cmd/runtimetest/main.go [hunk 2]
```diff
@@ -17,6 +17,10 @@ import (
 	"github.com/syndtr/gocapability/capability"
 )
 
+// PR_GET_NO_NEW_PRIVS isn't exposed in Golang so we define it ourselves copying the value
```
Comment (mrunalp) [inline cmd/runtimetest/main.go]: the value from..

Comment (Mashimiao) [inline cmd/runtimetest/main.go]: updated, thanks

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**